web_external/layout/layoutHeader.jade

```diff
@@ -82,6 +82,11 @@ nav.navbar.navbar-default.navbar-fixed-top.isic-header-wrapper
                 a(title="Upload Dataset", href="#uploadDataset")
                   i.icon-upload
                   | Upload Dataset
+            if currentUser
```

Comment:
Let's only render this if the current user has upload permissions too. There no need to confuse most users with 2 places to get to the form to request contributor access.